Makefile

```diff
@@ -87,6 +87,10 @@ endif
 GO_BUILD = CGO_ENABLED=0 GOARCH=$(GOARCH) $(GO) build $(GO_BUILD_FLAGS)
 GO_BUILD_HOOK = CGO_ENABLED=0 GOARCH=$(GOARCH) $(GO) -C contrib/rthooks/tetragon-oci-hook build $(GO_BUILD_FLAGS)
 
+.PHONY: kind
+kind:
+	./contrib/localdev/bootstrap-kind-cluster.sh
+
```

Comment:
I would not put this as the first rule since it will be triggered by running `make` instead of the `all` target.